### PR TITLE
address warnings

### PR DESCRIPTION
**Pull request by IvanIvanoff**
- **Do not add parentheses when struct field is accessed**
- **Mark non-secrets more explicitly**
- **Remove Endpoint confing from prod.exs to avoid warnings**
- **Fix warnings reported by credo**

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have tried to find clearer solution before commenting hard-to-understand parts of code
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: config/config.exs [hunk 1]
```diff
@@ -140,15 +140,15 @@ config :sanbase, Sanbase.Accounts.Hmac, secret_key: {:system, "APIKEY_HMAC_SECRE
 config :sanbase, SanbaseWeb.Endpoint,
   http: [protocol_options: [max_request_line_length: 16_384, max_header_value_length: 8192]],
   url: [host: "localhost"],
-  secret_key_base: "not_secret_Vq7Rfo0T4EfiLX2/ryYal3O0l9ebBNhyh58cfWdTAUHxEJGu2p9u1WTQ31Ki4Phj",
+  secret_key_base: "not_secret_please_do_not_report",
```
Comment (tspenov) [inline config/config.exs:143]: there are errors in the tests that say that is should be at lest 64 bytes

File: config/prod.exs [hunk 2]
```diff
@@ -1,30 +1,5 @@
...
-  cache_static_manifest: "priv/static/cache_manifest.json",
-  root: ".",
-  version: Application.spec(:sanbase, :vsn),
-  load_from_system_env: true,
-  check_origin: [
-    "//*.santiment.net",
-    "//*.sanr.app",
-    "//*.sanbase-admin.stage.san",
-    "//*.sanbase-admin.production.san"
-  ]
```
Comment (tspenov) [inline config/prod.exs:26]: Is this config not needed ?

Comment (IvanIvanoff) [reply config/prod.exs:26]: We had config in:
- prod.exs
- runtime.exs
- endpoint.ex

Now we only have left the one in `runtime.exs`.
The `SanbaseWeb.Endpoint.init` did have the highest precedence at the moment, I think. But it is deprecated:
```elixir
warning: SanbaseWeb.Endpoint.init/2 is deprecated, use config/runtime.exs instead or pass additional options when starting the endpoint in your supervision tree
  (phoenix 1.7.14) lib/phoenix/endpoint/supervisor.ex:36: Phoenix.Endpoint.Supervisor.init/1
  (stdlib 5.2) supervisor.erl:330: :supervisor.init/1
  (stdlib 5.2) gen_server.erl:980: :gen_server.init_it/2
  (stdlib 5.2) gen_server.erl:935: :gen_server.init_it/6
  (stdlib 5.2) proc_lib.erl:241: :proc_lib.init_p_do_apply/3
  ```

File: config/config.exs (hunk 1 above)
Comment (IvanIvanoff) [reply config/config.exs]: yep, fixed

**Review by tspenov — APPROVED** [review]

**Pull request merged**